requirements.txt

```
@@ -1,5 +1,3 @@
-discord.py >= 1.6
-mariadb >= 1.0.5
+discord.py >= 1.7
```

```suggestion
discord.py >= 1.7.1
```
patch has been added